src/Config_Command.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use EE\Model\Site;
+use Mustangostang\Spyc;
+use \Symfony\Component\Filesystem\Filesystem;
+
+/**
+ * Manges global EE configuration.
+ *
+ * ## EXAMPLES
+ *
+ *     # Save value in config
+ *     $ ee config set le-mail='abc@example.com' admin-email=abcd@example1.com
+ *
+ *     # Get value from config
+ *     $ ee config get le-mail
+ *
+ * @package ee-cli
+ */
+class Config_Command extends EE_Command {
+
+	/**
+	 * @var Filesystem $fs Symfony Filesystem object.
+	 */
+	private $fs;
+
+	public function __construct() {
+
+		$this->fs = new Filesystem();
+	}
+
+	/**
+	 * Get a config value
+	 *
+	 * ## OPTIONS
+	 *
+	 * <config-key>
+	 * : Name of config value to get
+	 */
+	public function get( $args, $assoc_args ) {
+		$config_file_path = getenv( 'EE_CONFIG_PATH' ) ? getenv( 'EE_CONFIG_PATH' ) : EE_CONF_ROOT . '/config.yml';
+		$config = Spyc::YAMLLoad( $config_file_path );
+
+		if ( ! isset( $config[$args[0]] ) ) {
+			EE::error("No config value with key '$args[0]' set");
+		}
+
+		EE::log( $config[$args[0]] );
+	}
+
+	/**
+	 * Get a config value
```

Comment:
Comment is not related to functionality. Please review your PR atleast once before you send it to the reviewer. You would've see this issue if you've review.

Reply:
I did review it, but I missed it.

Reply:
Done